lib/Models/Leaflet.ts

```diff
@@ -63,7 +63,7 @@ import Terria from "./Terria";
 // We want TS to look at the type declared in lib/ThirdParty/terriajs-cesium-extra/index.d.ts
 // and import doesn't allows us to do that, so instead we use require + type casting to ensure
 // we still maintain the type checking, without TS screaming with errors
-const FeatureDetection: FeatureDetection =
+const _FeatureDetection: FeatureDetection =
```

Comment:
This looks like something special that isn't just an unused parameter, can `@ts-expect-error` be used here so this variable doesn't get lost among the noise of unused parameters?

Reply:
Scratch my previous remark, the comment in the source code is 4+ years old. In #6944, the FeatureDetection variable is removed from CorsProxy.ts since the FeatureDetection is not used in that file any longer, so this line corresponds to an unused import as far as I understand. I'm guessing the variable and the comment can be removed?